main.py

```diff
@@ -52,27 +54,31 @@ def __init__(self, **kwargs):
             intents=discord.Intents.all(),
             case_insensitive=True,
             allowed_mentions=discord.AllowedMentions(everyone=False, roles=False),
+            activity=discord.Game(name='use the prefix "tim."'),
             **kwargs,
         )
-        self.session = ClientSession(loop=self.loop)
         self.start_time = datetime.datetime.utcnow()
         self.clean_text = commands.clean_content(escape_markdown=True, fix_channel_mentions=True)
 
     """  Events   """
 
-    async def on_connect(self):
+    async def setup_hook(self) -> None:
         """Connect DB before bot is ready to assure that no calls are made before its ready"""
+        self.session = ClientSession(loop=self.loop)
         self.db = await DataBase.create_pool(bot=self, uri=settings.postgres.uri, loop=self.loop)
+        for ext in initial_cogs:
+            try:
+                await self.load_extension(ext)
+            except Exception as error:
+                log.error(f"Failed to load extension {ext!r}:")
+                traceback.print_exception(type(error), error, error.__traceback__)
+
+        log.info(f"Loaded all extensions after {human_timedelta(self.start_time, brief=True, suffix=False)}")
 
     async def on_ready(self):
-        print(f"Successfully logged in as {self.user}\nSharded to {len(self.guilds)} guilds")
+        log.info(f"Successfully logged in as {self.user}. Sharded to {len(self.guilds)} guilds")
         self.guild = self.get_guild(settings.guild.id)
         self.welcomes = self.guild.get_channel(settings.guild.welcomes_channel_id)
-        await self.change_presence(activity=discord.Game(name='use the prefix "tim."'))
```

Comment:
Since we're removing this here, we should add a daily task that updates the presence.
From my experience I've seen that when you only set the presence when the bot starts, it will disappear after time.